app/components/linkable-header.hbs

```diff
@@ -0,0 +1,6 @@
+<h2 id={{@linkTarget}} class="heading heading--linkable" tabindex="0">
+    {{@linkTitle}}
+  <a title={{@linkTitle}} href='#{{@linkTarget}}' class="heading__link">
```

Comment:
I wonder if it's worth simplifying the component API even further by letting the component parse the title into a URL friendly link rather than having the developer have to think about that.

Reply:
Good thinking. I just pushed an update to do that.

Reply:
Nice! Thanks!